src/app/series/series.routing.ts

```diff
@@ -3,22 +3,42 @@ import { Routes, RouterModule } from '@angular/router';
 import { AuthGuard } from '../shared';
 
 import { SeriesComponent } from './series.component';
+import { SeriesBasicComponent } from './directives/series-basic.component';
+import { SeriesImageComponent } from './directives/series-image.component';
+import { SeriesTemplatesComponent } from './directives/series-templates.component';
+import { SeriesAdvancedComponent } from './directives/series-advanced.component';
 
 export const seriesRoutes: Routes = [
   {
-    path: 'series/create',
+    path: 'series/:id',
     component: SeriesComponent,
-    canActivate: [AuthGuard]
+    canActivate: [AuthGuard],
+    children: [
+      { path: '',          component: SeriesBasicComponent },
+      { path: 'image',     component: SeriesImageComponent },
+      { path: 'templates', component: SeriesTemplatesComponent },
+      { path: 'advanced',  component: SeriesAdvancedComponent }
+    ]
   },
   {
-    path: 'series/:id',
+    path: 'series/new',
```

Comment:
Should this route be above :id since it's more specific? Looks like you may not even need the new route as a separate route since the route configs are basically the same.